### PR TITLE
fix: update azure api_endpoint and readme

### PR DESCRIPTION
**Pull request by wavemotionio**
PR resolves https://github.com/dependabot/dependabot-script/issues/362

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ There are other variables that you must pass to your container that will depend
 * Azure DevOps
     * AZURE_HOSTNAME: default value `dev.azure.com`
     * AZURE_ACCESS_TOKEN
+    * PROJECT_PATH=organization/project/_git/package-name
```
Comment (greysteil) [inline README.md]:
```suggestion
    * PROJECT_PATH: (i.e., organization/project/_git/package-name)
```

**Comment by greysteil** [comment]
Thanks!

File: README.md (hunk 1 above)
Comment (wavemotionio) [reply README.md]: Sure!

Comment (greysteil) [reply README.md]: You were too fast for me!

**Pull request merged**